api/routes/user_routes.py

```diff
@@ -160,3 +193,47 @@ def get_all_associations_assigned_by_user(assigner_id, active=None):
 
     current_app.logger.error(f"Could not find any applications assigned by user {assigner_id}")
     abort(404)
+
+
+def send_notification_email(application_id, user_id, assigner_id, template, message=None):
+    """Sends a notification email to inform the user (specified by user_id) that
+    an application has been assigned to them.
+
+    Parameters:
+        application_id (str): id of application that has been assigned
+        user_id (str): id of assignee and recipient of email
+        assigner_id (str): id of the assigner.
+        template (str): template of email (either assignment or unassignment)
+        message (str): Custom message provided by assigner
+
+    """
+    application = get_metadata_for_application(application_id)
+    user_response = get_data(Config.ACCOUNT_STORE_API_HOST + Config.ACCOUNTS_ENDPOINT, {"account_id": user_id})
```

Comment:
Using `get_data` feels quite high level at the route controller level, I wonder if this should be delegated to a util/ service specifically for the account/ fund store? It might be easier to cover this behaviour if needed in unit/ contract tests.

Appreciate this is how its done in other places so definitely shouldn't slow this PR down

Reply:
Nice looks like this is well address in https://github.com/communitiesuk/funding-service-design-assessment-store/pull/496/commits/cd6831289b470df500ecabc0d8c8b09d3cce988e